auth/templates/uniform/allauth/account/signup.html

```diff
@@ -143,15 +141,28 @@
 document.getElementById('div_id_password2').children[0].textContent='PASSWORD (AGAIN)';
 
 
-function hideCourseCode(){
+function showInstructorForm(){
+    var message = "In the instructor account, instructors set up and customize " +
+            "their own StarCellBio assignments and experiments. The email address " +
+            "for the instructor account must be different from those used to set " +
+            "up other StarCellBio accounts. To set up an instructor account, enter " +
+            "your email and password below and then click Create Account.";
     document.getElementById('div_id_course_code').style.display = 'none';
+    document.getElementById('toggle_message').innerHTML = message;
 }
 
-function showCourseCode(){
+function showStudentForm(){
+    var message = 'Your instructor should have provided you with a code that you need ' +
+            'to enter below to register and use this StarCellBio tool.Please contact ' +
```

Comment:
Can you add a space at the end of the sentence at `... StarCellBio tool.Please contact `